docs/setup/upgrade.asciidoc

```diff
@@ -1,45 +1,53 @@
   You **cannot roll back** to an earlier version unless you have a backup of
   your data.
 * If you are using custom plugins, check that a compatible version is
   available.
 ===========================================
 
-Depending on which version of Kibana you're upgrading from, the upgrade process
-will vary. Please consult the following table to determine which process you
-should follow:
-
-[cols="1<m,1<m,3",options="header",]
-|=======================================================================
-|Upgrade From   |Upgrade To     |Supported Upgrade Type
-|3.x            |5.x            |<<upgrade-new-install,New installation>>
-|4.0 or 4.1     |5.x            |<<upgrade-standard-reindex,Standard upgrade & reindex>>
-|4.x >= 4.2     |5.x            |<<upgrade-standard,Standard upgrade>>
-|5.0.0 pre GA   |5.x            |<<upgrade-standard-reindex,Standard upgrade & reindex>>
-|5.x            |5.y            |<<upgrade-standard,Standard upgrade>> (where `y > x`)
-|5.x            |6.x            |<<upgrade-standard,Standard upgrade>>
-// & <<migrating-6.0-index,Reindex transformation>>
-|6.x            |6.y            |<<upgrade-standard,Standard upgrade>> (where `y > x`)
-|6.x            |7.x            |<<upgrade-standard,Standard upgrade>>
-|=======================================================================
-
-NOTE: Saved searches, visualizations, and dashboards created in Kibana 4.x or 5.x
-can be generally imported into 6.x.
+Depending on which version of {kib} you're upgrading from, the upgrade process to 7.0
+will vary. 
 
-include::upgrade/upgrade-standard.asciidoc[]
+The recommended path is to upgrade to 6.7 before upgrading to 7.0. 
+This makes it easier to identify the changes you need to make to upgrade and enables 
+you to perform a rolling upgrade with no downtime. 
+
+Please see 
+*{stack-ref}/upgrading-elastic-stack.html[Upgrading the Elastic Stack]* for a 
+comprehensive overview of the upgrade process.
+
+[float]
+==== Upgrading from 6.6 or earlier
+{es} can read indices created in the previous major version. If you have indices 
+created in 5.x or before, you must reindex or delete them before upgrading to 7.0.0.
+See {stack-ref}/upgrading-elastic-stack.html#oss-stack-upgrade[Upgrading from 6.6 or earlier] 
+for more information.
+
+Once your reindex is complete, you can follow the <<upgrade-standard, Standard upgrade>> 
+instructions to do a manual upgrade. The recommended path is to upgrade to 6.7 before upgrading to 7.0.
 
-include::upgrade/upgrade-standard-reindex.asciidoc[]
+*Note:* Saved searches, visualizations, and dashboards created in {kib} 6.x
+can be generally imported into 7.x.
 
-include::upgrade/upgrade-new-install.asciidoc[]
+[float]
+==== Upgrading from 6.7
+6.7 includes an https://www.elastic.co/guide/en/kibana/6.7/upgrade-assistant.html[Upgrade Assistant] 
+to help you prepare for your upgrade to 7.0. To access the assistant, go to *Management > 7.0 Upgrade Assistant*. 
+
```

Comment:
FYI: The last step in the Upgrade Assistant is a referral to these upgrading docs:
![image](https://user-images.githubusercontent.com/26471269/55244570-a7301500-51fe-11e9-9bd0-9abe2b52a966.png)
That is to say, it does not do the upgrade for you.  Therefore I think the "You can also..." sentence incorrectly implies that you either use the Upgrade Assistant OR do the upgrade manually.

Reply:
Good point, thanks